meetings/2017-09-13.md

```diff
@@ -0,0 +1,151 @@
+* Copyright Date [#195](https://github.com/nodejs/TSC/issues/195)
+* Regular roll-up reporting for active working groups [#109](https://github.com/nodejs/TSC/issues/109)
+
+### nodejs/node
+
+* stream: remove {writeableState/readableState}.length [#12857](https://github.com/nodejs/node/pull/12857)
+* Cleanup `_writableState` and `_readableState` access across codebase [#445](https://github.com/nodejs/node/issues/445)
+
+### nodejs/CTC
+
+* Requesting Rod to step down or be removed from the CTC [#165](https://github.com/nodejs/CTC/issues/165)
+
+## Announcements
+
+No announcements this week.
+
+## Previous Meeting Review
+
+We skipped this as we had a packed agenda.
+
+## Minutes
+
+### nodejs/TSC
+* New TSC Director/Chair Elections [#326](https://github.com/nodejs/TSC/issues/326)
+ * No objections, Myles acclaimed as Directory, Michael acclaimed as Chair
+
+* TSC Meeting Schedule [#336](https://github.com/nodejs/TSC/issues/336)
+ * We should revisit quarterly, ideally aligned with 
+   daylight savings times changes.
+ * Agreed we should move forward with the proposed times.
+
+* 2FA for the entire org [#301](https://github.com/nodejs/TSC/issues/301)
+  * We have consensus on step 1, will take of agenda.
+  * Myles, explore usb keys, could foundation fund ?
+  * Rich, is has not been an issue so far, can we defer until we see
+    if it will be an issue.
+  * Matteo, biggest issue is telling and reminding people to do it.
+  * Matteo, would be good to have  policy that if you need financial
+    support you can request it.
+
+* Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)
+  * Michael gave overview and request to get involved in issue
+  * Mateo highlighted Metrics as one where we might do more.
+
+* Copyright Date [#195](https://github.com/nodejs/TSC/issues/195)
+  * Myles to bring up at one board meeting
+  * Remove from TSC agenda.
+
+* Regular roll-up reporting for active working groups [#109](https://github.com/nodejs/TSC/issues/109)
+  * Myles is going to try to get podcast style updates going.
+
+
+### nodejs/node
+
+* stream: remove {writeableState/readableState}.length [#12857](https://github.com/nodejs/node/pull/12857)
+  * tied to 445, see discussion there
+
+* Cleanup `_writableState` and `_readableState` access across codebase [#445](https://github.com/nodejs/node/issues/445)
+  * prefer not to tack on properties to people’s object
+  * having bundled into separate objects is better
+  * anna, Brian are you ok with current state ?  
+  * Brian, _ not ideal but if we can’t agree on any alternatives yes
+  * Can’t explain how it works without including discussions of
+    internal state machine.
+  * If we have separate object that refers to existing one, but
+    Matteo believes this may be an issue due to additional allocation
+  * Jeremiah just consider _XX for writable and readable part of 
```

Comment:
Nit: I think the `_` has to be escaped here.

Reply:
I looked at in "view" and it seemed ok.